clients/src/main/java/org/apache/kafka/common/requests/FetchMetadata.java

```diff
@@ -114,9 +114,21 @@ public boolean equals(Object o) {
     }
 
     /**
-     * Return the metadata for the next error response.
+     * Return the metadata for the next request. The metadata is set to indicate that the client wants to close the
+     * existing session.
      */
     public FetchMetadata nextCloseExisting() {
+        return new FetchMetadata(sessionId, FINAL_EPOCH);
+    }
+
+    /**
+     * Return the metadata for the next request.
```

Comment:
Should we just just say `The metadata is set to indicate that the client wants to close the existing session and create a new one if possible.`?

Reply:
sure, this is much clearer. Done in upcoming commit.